controller/src/main/java/io/pravega/controller/server/eventProcessor/RequestUnsupportedException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.eventProcessor;
+
+import io.pravega.controller.server.ControllerServerException;
+
+/**
+ * Unsupported Request thrown when there is no requestHandler to process the request.
+ */
+public class RequestUnsupportedException extends ControllerServerException {
```

Comment:
Instead of creating a new exception type for each new error condition, we could encode different error conditions by different values of `error` field of `ControllerServerException`?